plugins/ipam/host-local/host_local_test.go

```diff
@@ -281,7 +281,7 @@ var _ = Describe("host-local Operations", func() {
 		ipFilePath := filepath.Join(tmpDir, "mynet", result.IPs[0].Address.IP.String())
 		contents, err := ioutil.ReadFile(ipFilePath)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(string(contents)).To(Equal("dummy"))
+		Expect(string(contents)).To(Equal("dummy" + "\n" + ifname))
 
 		// Release the IP
 		err = testutils.CmdDelWithArgs(args, func() error {
```

Comment:
I'd add two more tests to this file that (a) allocates two IPs for the same container on different interfaces, then deletes one and makes sure the right file got deleted, and (b) writes an old-style (eg containerid only) IP file to the data dir and runs DEL and makes sure the old-style file is deleted.